src/storage/backend/sqlite/SQLite.cc

```diff
@@ -43,41 +91,39 @@ OperationResult SQLite::DoOpen(OpenResultCallback* cb, RecordValPtr options) {
         return open_res;
     }
 
-    std::string create = "create table if not exists " + table_name + " (";
-    create.append("key_str blob primary key, value_str blob not null, expire_time real);");
-
     char* errorMsg = nullptr;
-    if ( int res = sqlite3_exec(db, create.c_str(), NULL, NULL, &errorMsg); res != SQLITE_OK ) {
-        std::string err = util::fmt("Error executing table creation statement: %s", errorMsg);
-        Error(err.c_str());
-        sqlite3_free(errorMsg);
-        Close(nullptr);
-        return {ReturnCode::INITIALIZATION_FAILED, std::move(err)};
-    }
 
-    if ( int res = sqlite3_exec(db, "pragma integrity_check", NULL, NULL, &errorMsg); res != SQLITE_OK ) {
-        std::string err = util::fmt("Error executing integrity check: %s", errorMsg);
-        Error(err.c_str());
-        sqlite3_free(errorMsg);
-        Close(nullptr);
-        return {ReturnCode::INITIALIZATION_FAILED, std::move(err)};
+    OperationResult pragma_res = RunPragma("integrity_check");
```

Comment:
One nice thing of the `pragma_commands` vector would be that `integrity_check` could be the first element, allowing users to skip it, too, without needing an extra option.

Reply:
Sounds reasonable.